lib/cmxl/field.rb

```diff
@@ -44,8 +44,8 @@ def self.tag
     # Cmxl::Field.parse(':60F:C031002PLN40000,00') #=> returns an AccountBalance instance
     #
     def self.parse(line)
-      if line.match(/^:(\d{2,2})(\w)?:(.*)$/)
-        tag, modifier, content = $1, $2, $3
+      if line.match(/\A:(\d{2,2})(\w)?:(.*)\z/m)
+        tag, modifier, content = $1, $2, $3.gsub(/\r?\n\z/, '') # gsub last line break
```

Comment:
can this be added to the regex somehow? maybe `\b\z` ?

Reply:
afaik this will not work as .* will consume everything
check this: http://rubular.com/r/rzwkQraNNR

Reply:
`.*` is a bitch. 
but isn't `\n` or something a word boundary before the end of the string?
what is this doing: http://rubular.com/r/EJ7DpCMq57